spec/loadbalancer/default.conf

```diff
@@ -1,27 +1,22 @@
 upstream master {
-     server 192.168.1.116:17990/agent;
-     server 192.168.1.105:17990/agent;
+     server IP1:17990;
```

Comment:
1) To use environment vars, it should be templitised as specified here https://serverfault.com/a/755541 This may break the image build. 
2) Please provide the **test build log**, if this believed to be a working setting.

Reply:
> Are the IP1/IP2 the env variables to the respective IP in the machine.env file? @ramaraosrikakulapu

@ayasuda2003 Thanks for review. Here are my comments - 
- Yes, IP1/IP2 are supposed to read from machine.env file
- Templatizing the conf file will taken care in future MR's. This MR includes, adding path based configuration and documentation

Reply:
Understand. Please propose a PR along with a test build log when the issue is resolved. The file is not usable in this container spec.

Reply:
Sure chia.. Will keep open the PR and will continue to take the suggestions

Reply:
default.conf is now templatized in latest commit. We just have to update IP addresses in in machine.env and pass as env-file to docker run command.